phantoms/MR_XCAT_qMRI/README.md

```diff
@@ -1 +1,61 @@
-# MR_XCAT_qMRI
\ No newline at end of file
+# 4D IVIM Phantom Generator
+
+A command-line tool for generating a 4D IVIM (Intravoxel Incoherent Motion) phantom as a NIfTI file.
+
+## Usage
+
+```sh
+python sim_vim_sig.py  [-h] [-b BVALUE [BVALUE ...] | -f FILE] [-n NOISE] [-m] [-i]
```

Comment:
Could you provide an abbreviated version of this help in the scrip itself? Perhaps with a `-h` help argument or if the expected inputs aren't provided.

Reply:
Do you meant this ? 

![Screenshot from 2024-03-12 21-43-29](https://github.com/OSIPI/TF2.4_IVIM-MRI_CodeCollection/assets/86585626/80b9a097-c3ad-47bc-aa72-78e024aa5dd2)


It is automatically, created.

Reply:
Oh, excellent.